### DISCUSSION TITLE
Updating version in `README.md` and other locations

### DISCUSSION THREAD

**Discussion by calteran** [release-plz/release-plz#1229] [Q&A] [answered]
Is there an accepted way to propagate the version number generated with `release-plz` into other project files?  My specific case is that my `README.md` includes the standard copy/paste-able line for adding my crate into someone's `Cargo.toml`.  It seems I could write a job to pick up the change when the PR is opened by `release-plz` and commit it in other files, but it would be cleaner to do that in the same commit `release-plz` generates.  If this isn't already a thing, I'd be happy to look into adding it via a PR, but I'd appreciate some direction on where to start.

**Comment by marcoieni** [comment, marked as answer]
Hi 👋 

> my README.md includes the standard copy/paste-able line for adding my crate into someone's Cargo.toml

Why don't you suggest to run `cargo add my_crate` instead?

Anyway, https://github.com/MarcoIeni/release-plz/issues/1024 is the issue that will solve this :)

https://github.com/MarcoIeni/release-plz-action/issues/88 would also help to detect the new version
